### ISSUE TITLE
Spellchecking: form controls don't have labels

### ISSUE THREAD

**Issue by dstorey** [MicrosoftEdge/Demos-old#236]
https://developer.microsoft.com/en-us/microsoft-edge/testdrive/demos/spellchecking/

They use placeholders instead but really should have labels with for/id. For the last two inputs, the span could be changed to a label for example.

**Comment by alrra** [comment]
Closing this in favor of #257.
